recode/base.py

```diff
@@ -466,19 +466,15 @@ def specs_from_frames(frames: Frames):
     >>> assert decoded_frames == [(1.1,2.2),(3.3,4.4)]
     """
     head, frames = spy(frames)
-
```

Comment:
Nothing. It's purely sytilstic, et probably a tiny bit more efficient. Instead of fetching `head[0]` 8 times, I do `head = head[0]` once and then use `head` itself.